installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -57,7 +57,7 @@ EOF
 vagrant_dev_eth1() {
     declare os=$1
     declare ip=$2
-    if [[ $os == fedora && $vagrant_dev_private_net ]]; then
```

Comment:
is `$os == fedora` true for alma?

Reply:
Most of the time. `$os` is guest os. So for sirepo and workers it is fedora. For rsconf primary it is (or will be) alma. `install_os` is host os. I'll rename `$os` to `$guest_os` to be clearer.